packages/compare-slider-block/src/CompareSliderBlock.tsx

```diff
@@ -0,0 +1,360 @@
+        if (hasRadius) {
+            return radiusValue;
+        }
+
+        return radiusStyleMap[radiusChoice];
+    };
+
+    const renderSliderItem = (slot: SliderImageSlot) => {
+        return (
+            <div>
+                <div style={{ background: hasBackground ? toRgbaString(backgroundColor) : undefined }}>
+                    <img
+                        src={slot === SliderImageSlot.First ? firstAssetPreviewUrl : secondAssetPreviewUrl}
+                        alt={slot === SliderImageSlot.First ? firstAssetTitle : secondAssetTitle}
+                        className="tw-w-full tw-object-cover"
+                        style={{ height: getImageHeight() }}
+                    />
+                </div>
+
+                {renderStrikethrough(slot)}
+                {!isEditing && renderLabel(slot)}
+            </div>
+        );
+    };
+
+    const renderStrikethrough = (slot: SliderImageSlot) => {
+        return (
+            <StrikethroughWrapper
+                alignment={alignment}
+                currentSliderPosition={currentSliderPosition}
+                hasStrikeThrough={
+                    slot === SliderImageSlot.First ? firstAssetHasStrikethrough : secondAssetHasStrikethrough
+                }
+                slot={slot}
+                borderRadius={getBorderRadius()}
+            >
+                <Strikethrough
+                    color={
+                        strikethroughColorSource === InheritSettings.OVERRIDE
+                            ? toRgbaString(customStrikeThroughColor)
+                            : designTokens.callout?.warning ?? toRgbaString(DEFAULT_STRIKETHROUGH_COLOR)
+                    }
+                />
+            </StrikethroughWrapper>
+        );
+    };
+
+    const renderLabel = (slot: SliderImageSlot) => {
+        const label = slot === SliderImageSlot.First ? firstAssetLabel : secondAssetLabel;
+        const key = slot === SliderImageSlot.First ? 'firstAssetLabel' : 'secondAssetLabel';
+
+        return (
+            <LabelWrapper
+                firstAssetLabelPlacementHorizontal={firstAssetLabelPlacementHorizontal}
+                firstAssetLabelPlacementVertical={firstAssetLabelPlacementVertical}
+                secondAssetLabelPlacementVertical={secondAssetLabelPlacementVertical}
+                secondAssetLabelPlacementHorizontal={secondAssetLabelPlacementHorizontal}
+                alignment={alignment}
+                slot={slot}
+                borderRadius={getBorderRadius()}
+            >
+                <Label
+                    isEditing={isEditing}
+                    designTokens={designTokens}
+                    value={label}
+                    onBlur={(newValue: string) => setBlockSettings({ [key]: newValue })}
+                />
+            </LabelWrapper>
+        );
+    };
+
+    if (isEditing && (!firstAsset || !secondAsset)) {
+        return (
+            <div className="tw-h-[500px] tw-flex">
```

Comment:
Good suggestion, I added it with a little tweak. If the height is set to auto then I will fallback to the medium value. (Because for auto calculation both images are needed)